src/debugger/android/androidPlatform.ts

```diff
@@ -19,17 +19,21 @@ import {Package} from "../../common/node/package";
 export class AndroidPlatform implements IAppPlatform {
     private extensionMessageSender: ExtensionMessageSender;
 
+    private static MULTIPLE_DEVICES_ERROR = "error: more than one device/emulator";
```

Comment:
My understanding was that this would not be an error case anymore. Why do we still need it?